Tools/unicode/makeunicodedata.py

```diff
@@ -1044,12 +1051,17 @@ def __init__(self, version, cjk_check=True):
                     if s.name.startswith("<CJK Ideograph"):
                         cjk_ranges_found.append((field[0],
                                                  s.codepoint))
+                    elif s.name.startswith("<Tangut Ideograph"):
+                        tangut_ranges_found.append((field[0],
+                                                    s.codepoint))
                     s.name = ""
                     field = None
             elif field:
                 table[i] = from_row(('%X' % i,) + field[1:])
         if cjk_check and cjk_ranges != cjk_ranges_found:
             raise ValueError("CJK ranges deviate: have %r" % cjk_ranges_found)
+        if cjk_check and tangut_ranges != tangut_ranges_found:
+            raise ValueError("Tangut ranges deviate: have %r" % tangut_ranges_found)
```

Comment:
Since the `cjk_check` flag is also used for checking the Tangut ranges under this PR's changes, it might make sense to rename it to `ideograph_check`. It's legible as written but slightly confusing since the flag advertises itself as related to CJK but the functionality exceeds that.

Reply:
Fair enough! Fixed.